mmv1/api/product/version.rb

```diff
@@ -22,13 +22,15 @@ class Product < Api::Object::Named
     class Version < Api::Object::Named
       include Comparable
 
+      attr_reader :validator_base_url
```

Comment:
I might have a slight preference for `cai_base_url` since it is format-specific rather than tool-specific; what do you think?